src/interactivity/win32/Clipboard.cpp

```diff
@@ -515,7 +275,9 @@ void Clipboard::CopyTextToSystemClipboard(const TextBuffer::TextAndColor& rows,
 
     if (fAlsoCopyHtml)
     {
-        std::string HTMLToPlaceOnClip = GenHTML(rows);
+        const auto& fontData = ServiceLocator::LocateGlobals().getConsoleInformation().GetActiveOutputBuffer().GetCurrentFont();
+        int const iFontHeightPoints = fontData.GetUnscaledSize().Y * 72 / ServiceLocator::LocateGlobals().dpi;

```

Comment:
wonder if there's an existing constant for this `72`.

Reply:
I hate to say it but I couldn't find anything on where the value came from or what constant we could use for it. :(